lib/config.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
```

Comment:
If I were to use a `config.js` file like this that is located in a directory outside of this repo, I suppose I would need to use npm to set up a peer dependency for React. Do you see another way to handle this, @evansiroky?

Reply:
I edited the webpack.config.js script to copy over that file into a temp folder in this repo. That way it can use any module installed for trimet-mod-otp. However, it won't be able to read in any other files that live outside the repo.